perf/reader.go

```diff
@@ -430,6 +438,11 @@ func (pr *Reader) Resume() error {
 	return nil
 }
 
+// Size is the size in bytes of each per-CPU buffer
+func (pr *Reader) Size() int {
```

Comment:
I think your `bufferSize` idea is better:

```suggestion
func (pr *Reader) BufferSize() int {
```